File: typescript/src/patterns/cqrs/Commands.ts
import { WriteModel } from './WriteModel.js';
export class Commands {
  constructor(private write: WriteModel) {}
  createUser(id: string, name: string, email: string) { this.write.createUser(id, name, email); }
  updateUserName(id: string, name: string) { this.write.updateUserName(id, name); }
}

File: python/src/patterns/cqrs/commands.py
from .write_model import WriteModel

class Commands:
    def __init__(self, write: WriteModel) -> None:
        self.write = write
    def create_user(self, id: str, name: str, email: str) -> None:
        self.write.create_user(id, name, email)
    def update_user_name(self, id: str, name: str) -> None:
        self.write.update_user_name(id, name)
